api/v1alpha1/openapi.yaml

```diff
@@ -3666,7 +3715,7 @@ components:
           $ref: '#/components/schemas/RolloutDeviceSelection'
         successThreshold:
           $ref: '#/components/schemas/Percentage'
-        defaultUpdateTimeout:
+        updateTimeout:
```

Comment:
I propose this change as its not technically the default meaning it can be changed by something else it is the `updateTimeout` this is aligned with UpdatePolicy.